examples/ant_colony/ant_colony/server.py

```diff
@@ -0,0 +1,42 @@
+import mesa
+
+from ant_colony.agents import Ant
+from ant_colony.model import AntColony
+from .SimpleContinuousModule import SimpleCanvas
+
+
+boid_canvas = SimpleCanvas(Ant, 500, 500)
+
+def ant_colony_portrayal(agent):
+    if agent is None:
+        return
+
+    portrayal = {}
+
+    if type(agent) is Ant:
+        portrayal["Shape"] = f"ant_colony/resources/{agent.color}_ant.png"
+        # https://icons8.com/icons/set/ant
+        portrayal["scale"] = 0.9
+        portrayal["Layer"] = 1
+
+    return portrayal
+
+canvas_element = SimpleCanvas(ant_colony_portrayal, 500, 500)
```

Comment:
I reduced the canvas size to 50, 50. And was able to run the simulation just fine.